Composer/packages/extensions/adaptive-form/src/components/fields/index.ts

```diff
@@ -16,3 +16,4 @@ export * from './RegexIntentField';
 export * from './SelectField';
 export * from './StringField';
 export * from './UnsupportedField';
+export * from './IntellisenseField';
```

Comment:
We should remove index files except at the package/extension/plug-in entry point.

Reply:
Should this be done as part of a separate PR? Otherwise many more files would be included in this PR..

Reply:
Sure.